client_code/ProgressBar/Indeterminate/__init__.py

```diff
@@ -18,12 +18,33 @@
 
 class Indeterminate(IndeterminateTemplate):
     def __init__(self, **properties):
+        self.indicator_dom_node = get_dom_node(self.indicator_panel)
         self.dom_node = get_dom_node(self)
         self._props = properties
         self.role = "ae-progress-track"
         self.indicator_panel.role = "ae-indeterminate-progress-indicator"
         self.init_components(**properties)
 
+    @property
+    def height(self):
+        return self._height
+
+    @height.setter
+    def height(self, value):
+        value = _css_length(value)
+        self._height = value
+        self.indicator_dom_node.style.setProperty("height", value)
```

Comment:
Better to convert to css length after we assign it to the private variable. That way the value you set is the value you get. This will also mean the designer property editors don't mysteriously change which I think they would currently. If you set the value to 10 then the designer might display 10px.

Reply:
Fair enough, I'll do that in the next push.